### PR TITLE
Issue 500 missing yaml prefixes

### PR DESCRIPTION
**Pull request by RichardBruskiewich**
Missing prefixes resolved as best  as  feasible

### CODE REVIEW

File: biolink-model.yaml [hunk 1]
```diff
@@ -4,24 +4,30 @@ description: Entity and association taxonomy and datamodel for life-sciences dat
 license: https://creativecommons.org/publicdomain/zero/1.0/
 
 # Version should be kept in sync with primary Git repository release tag
-version: 1.4.0
+version: 1.4.1
```
Comment (deepakunni3) [inline biolink-model.yaml:7]: Still a 1.4.0

**Review by deepakunni3 — CHANGES_REQUESTED** [review]

**Comment by deepakunni3** [comment]
Couple of issues that could be addressed as part of this PR:
- `oboformat` prefix defined but curies have `oboFormat`
- https://github.com/biolink/biolink-model/blob/ca7d1f666cfdbb4941a13748afdc4c69f76e78b0/biolink-model.yaml#L608
  - Prefix here should be NCIT instead of CIT
- `isbn` and `issn` prefix definition missing

**Comment by RichardBruskiewich** [comment]
> Couple of issues that could be addressed as part of this PR:
> 
>     * `oboformat` prefix defined but curies have `oboFormat`

OK, I'll  fix  this  and  commit the  fix in a few minutes.
> 
>     * https://github.com/biolink/biolink-model/blob/ca7d1f666cfdbb4941a13748afdc4c69f76e78b0/biolink-model.yaml#L608
>       
>       
>       * Prefix here should be NCIT instead of CIT

ditto
> 
>     * `isbn` and `issn` prefix definition missing

Actually, no  they  aren't: they  are  in  the context.jsonld  (I  think, imported  from  prefixcommons.org)

**Comment by cmungall** [comment]
after this can we switch to a stricter checking mode such that it's
impossible to introduce new curies with undeclared prefixes rather than
relying on manual inspection?

would be good to have a prefix report tsv, or a github page per prefix...

**Comment by deepakunni3** [comment]
@cmungall I agree. Would be useful to have the reports. I'll make a ticket for that

**Comment by RichardBruskiewich** [comment]
* `isbn` and `issn` prefix definition missing

I was mistaken for some reason in believing that prefixcommons.org prefixes for these were propagating into the model so I put them in

**Pull request merged**